app/models/station.rb

```diff
@@ -0,0 +1,3 @@
+class Station < ApplicationRecord
+  validates_presence_of :name, :dock_count, :city, :installation_date
+end
```

Comment:
@jamesrnelson @jrambold @hmesander should we validate all the fields? Or will we have incomplete data?

Reply:
I think we should validate all of the fields now and deal with defaults for incomplete data when we seed the db.